src/main/webapp/i18n/de/result.json

```diff
@@ -71,7 +71,8 @@
                 }
             },
             "notLatestSubmission": "Hinweis: Dieses Ergebnis und Feedback bezieht sich <strong>nicht</strong> auf die letzte Einreichung.",
-            "seeMore": "Mehr anzeigen"
+            "seeMore": "Mehr anzeigen",
+            "afterDueDateFeedbackHidden": "Auch nach der Abgabefrist werden nicht immer Detailfeedbacks zu allen Testfällen angezeigt. Dies ist etwa der Fall wenn für andere Studierende eine spätere Abgabefrist gesetzt ist (z.B. zum Ausgleich von Krankheitstagen). Sobald alle Abgaben erfolgt sind, werden auch diese Feedbacks automatisch sichtbar."
```

Comment:
```suggestion
            "afterDueDateFeedbackHidden": "Auch nach der Abgabefrist werden nicht immer Detail-Feedbacks zu allen Testfällen angezeigt. Dies ist etwa der Fall, wenn für andere Studierende eine spätere Abgabefrist gesetzt ist (z.B. zum Ausgleich von Krankheitstagen). Sobald alle Abgaben erfolgt sind, werden auch diese Feedbacks automatisch sichtbar."
```

Reply:
Both ‘Detail’ and ‘Feedback’ are nouns present in the Duden dictionary. Therefore, I prefer the using a proper compound-noun without the hyphen.